docs/deployment.md

```diff
@@ -81,7 +81,23 @@ Secret keys are used to encrypt cookies, password reset emails [and other things
 openssl rand -hex 32
 ```
 
-# K8s
+# Helm Charts
+
+We maintain a [helm chart for PostHog](https://github.com/PostHog/charts/tree/master/charts/posthog).
+
+To install the latest version:
+
+```shell script
+helm repo add posthog https://posthog.github.io/charts/
+helm repo update
+helm install posthog posthog/posthog
+```
+
+See the [README](https://github.com/PostHog/charts/blob/master/charts/posthog/README.md) or 
+[`values.yaml`](https://github.com/PostHog/charts/blob/master/charts/posthog/values.yaml)
+for configuration options.
+
+# Raw K8s
```

Comment:
Do we still need raw K8s config if we have helm chart? I think it's also out of date (no workers for example) so in favour of just moving to charts

Reply:
Good point. It's definitely out of date. Removed. If anyone wants it back, I'm sure we'll hear about it.